packages/core/test/tools/mocky.js

```diff
@@ -35,20 +35,18 @@ const fakeSignedPostData = {
     AWSAccessKeyId: 'AKIAIKIAAKIAIKIAAKIA',
     acl: 'public-read',
     key: 'some-route/d362f087-1106-4847-9261-669ec340b580',
-    signature: 'c4GzkaCtrc0ruvbZh6aSmf/1k='
-  }
+    signature: 'c4GzkaCtrc0ruvbZh6aSmf/1k=',
+  },
 };
 
-const mockUpload = bodyMatcher => {
-  nock('https://s3-fake.zapier.com')
-    .post('/', bodyMatcher)
-    .reply(204, '');
```

Comment:
removing the `''` was the only actual change here. That's the default in `nock` now, so I didn't figure we needed it.